src/UI/templates/default/Button/tpl.shy.html

```diff
@@ -0,0 +1 @@
+<a class="btn btn-link<!-- BEGIN disabled --> ilSubmitInactive<!-- END disabled -->"<!-- BEGIN with_href --> href="{HREF}"<!-- END with_href --> data-action="{ACTION}"<!-- BEGIN with_id --> id="{ID}"<!-- END with_id -->>{LABEL}</a>
```

Comment:
btn-link is an existing bootstrap class for this purpose, btn-shy is not:
http://getbootstrap.com/css/#buttons-options